userprefs.php

```diff
@@ -932,18 +917,18 @@ function _show_credits_wanted_adhoc(): void
     $pm_credit_checked = $userSettings->get_boolean('pm_anonymous') ? '' : 'checked ';
     $pp_credit_checked = $userSettings->get_boolean('pp_anonymous') ? '' : 'checked ';
 
-    echo "<input type='checkbox' name='cp_credit' value='yes' $cp_credit_checked> CP\n";
-    echo "<input type='checkbox' name='ip_credit' value='yes' $ip_credit_checked> IP\n";
-    echo "<input type='checkbox' name='tp_credit' value='yes' $tp_credit_checked> TP\n";
+    echo "<input type='checkbox' name='cp_credit' class='credit_box' value='yes' $cp_credit_checked> CP\n";
```

Comment:
minor suggestion, but instead of adding a class to every checkbox, maybe just tag the table cell and then query all inputs under that cell?
```
tableCell.querySelectorAll('input[type=checkbox]')
```

Reply:
Good idea. I'll look into it.

Reply:
Is @chrismiceli's suggestion still being considered?

Reply:
Thanks for reminding me. @chrismiceli's suggestion would make it more compact but looking at structure of userprefs.php, the function _show_credits_wanted_adhoc() is far removed from the table structure so I think it makes it clearer how it is intended to work to leave it like it is. (Leaving aside the unusual way the show_ functions are called, see comment on line 874).